### DISCUSSION TITLE
How to use OnPress in TabBar's renderLabel

### DISCUSSION THREAD

**Discussion by EricaLi123** [react-native-community/react-native-tab-view#1281] [Q&A] [answered]
Why does not Clicking btn1 or btn2 trigger event of onPress?

      <TabBar
                renderLabel={() => (
                    <View>
                        <Button onPress={() => console.log('btn1')} title={'btn1'} />
                        <Button onPress={() => console.log('btn2')} title={'btn2'} />
                    </View>
                )}
            />

**Comment by edreyyo** [comment, marked as answer]
The `onTabPress` event handler is likely getting fired instead. See https://github.com/satya164/react-native-tab-view#ontabpress

**Reply by EricaLi123** [reply]
But how can I tell if the clicked element is btn1 or btn2?
